modules/tasksUI.js

```diff
@@ -0,0 +1,131 @@
+import _ from 'lodash'; // eslint-disable-line no-unused-vars
+// import Task from './taskClass.js';
+
+export default class TasksUI {
+  static checkData = () => {
+    // console.log('checking database')
+    // console.log(localStorage.getItem('tasksDB') == '[]')
+    if (
+      localStorage.getItem('tasksDB') === null
+      || localStorage.getItem('tasksDB') === '[]'
+    ) {
+      // console.log(localStorage.getItem('tasksDB'));
+      return false;
+    }
+    return true;
+  };
+
+  static loadTasksUI = () => {
+    // console.log(this.checkData());
+    // console.log('dataBase checking before loading Tasks UI')
+    if (this.checkData() === false || this.checkData() === null) {
+      const tasksUIelement = document.querySelector('#taskUI');
+      tasksUIelement.style = 'height: 10%;';
+      tasksUIelement.innerText = 'Please add ani friends to interact';
+      tasksUIelement.className = 'warn-message flex';
+    } else {
+      const tasksUIelement = document.querySelector('#taskUI');
+      tasksUIelement.style = 'height: 80%;';
+      tasksUIelement.innerText = '';
+      tasksUIelement.className = '';
+      // console.log('you have data on your LcStr');
+      const tasksDbInstance = JSON.parse(localStorage.getItem('tasksDB'));
+      tasksDbInstance
+        // .filter((task) => task.completed !== true)
+        .forEach((task) => {
+          // console.log(task);
+          const taskContainer = document.createElement('div');
+          taskContainer.className = 'list-element flex';
+          taskContainer.id = `${task.id}`;
+          tasksUIelement.appendChild(taskContainer);
+          const taskToogle = document.createElement('input');
+          taskToogle.type = 'checkbox';
+          taskToogle.className = 'task';
+          taskToogle.checked = false;
+          taskToogle.id = `toogle${task.id}`;
+          taskContainer.appendChild(taskToogle);
+          const taskTitle = document.createElement('input');
+          taskTitle.className = 'task-title-text';
+          if (task.completed) {
+            taskToogle.checked = true;
+            taskTitle.style = 'text-decoration-line: line-through;';
+          } else {
+            taskToogle.checked = false;
+            taskTitle.style = '';
+          }
+          taskTitle.id = `title${task.id}`;
+          taskTitle.value = task.description;
+          taskContainer.appendChild(taskTitle);
+
+          taskTitle.addEventListener('change', (event) => {
+            if (taskTitle.value !== task.description) {
+              event.preventDefault();
+              // console.log('pressed Enter');
+              this.editTask(event);
+            }
+          });
+
+          taskToogle.addEventListener('change', (event) => {
+            // console.log(event);
+            // console.log(event.target.id);
+            // const order = event.target.id - 1
```

Comment:
kindly do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.